From 0d84e5885463eb2e9b85a67be6f1151efa3af881 Mon Sep 17 00:00:00 2001
From: Guan Jing <guanjing6@huawei.com>
Date: Sat, 2 Apr 2022 11:49:43 +0800
Subject: [PATCH 0877/2065] [Huawei] config: enable
 CONFIG_QOS_SCHED_SMT_EXPELLER by

Offering: HULK
hulk inclusion
category: feature
bugzilla: 186300

--------------------------------

Signed-off-by: Guan Jing <guanjing6@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index a4fd4cdcbd3e..9dc1be9dcb9d 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -151,6 +151,7 @@ CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_PERF=y
 CONFIG_CGROUP_BPF=y
 CONFIG_QOS_SCHED=y
+CONFIG_QOS_SCHED_SMT_EXPELLER=y
 CONFIG_QOS_SCHED_DYNAMIC_AFFINITY=y
 # CONFIG_CGROUP_DEBUG is not set
 CONFIG_SOCK_CGROUP_DATA=y
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 918de84d99f3..9c107d4c11f7 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -157,6 +157,7 @@ CONFIG_BLK_CGROUP=y
 CONFIG_CGROUP_WRITEBACK=y
 CONFIG_CGROUP_SCHED=y
 CONFIG_QOS_SCHED=y
+CONFIG_QOS_SCHED_SMT_EXPELLER=y
 CONFIG_QOS_SCHED_DYNAMIC_AFFINITY=y
 CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_CFS_BANDWIDTH=y
-- 
2.17.1

